front/src/common/Map/BaseMap.tsx

```diff
@@ -134,6 +136,7 @@ const BaseMap = ({
           infraId={infraId}
           mapStyle={mapStyle}
           hoveredOperationalPointId={hoveredOperationalPointId}
+          compact={compact}
```

Comment:
Instead of compact, you should give here the layerSettings directly
```suggestion
          layerSettings={layerSettings}
```

(Modify the BaseMapProps like this:)
```
type MapProps = Pick<MapState, 'layersSettings' | 'mapSearchMarker' | 'mapStyle' | 'showOSM'> & {...}
```